src/Nethermind/Nethermind.WebSockets/ISocketsClient.cs

```diff
@@ -19,12 +19,11 @@
 
 namespace Nethermind.WebSockets
 {
-    public interface IWebSocketsClient
+    public interface ISocketsClient : IDisposable
```

Comment:
Can you add documentation comment explaining what it is (especially one can confuse ISocketHandler with ISocketClient)